change/@ni-nimble-components-c9f2f270-d0d3-478a-b43a-c0777af5fc44.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Could you update the component status table too? I think we use ⚠️ for in-work components.

Reply:
Updated